docs/api-reference/next/image.md

```diff
@@ -43,17 +45,101 @@ function Home() {
 export default Home
 ```
 
-`Image` accepts the following props:
+## Props
+
+The `Image` component accepts the following properties.
+
+### src
+
+The path or URL to the source image. This is required.
+
+### width
+
+The width of the image, in pixels. Must be an integer without a unit.
+
+Required unless [layout="fill"`](#layout).
+
+### height
+
+The height of the image, in pixels. Must be an integer without a unit.
+
+Required unless [layout="fill"`](#layout).
+
+### layout
+
+The layout behavior of the image as the viewport changes size.
+
+When `fixed`, the image dimensions will not change as the viewport changes (no responsiveness) similar to the native `img` element.
+
+When `intrinsic`, the image will scale the dimensions down for smaller viewports but maintain the original dimensions for larger viewports.
+
+When `responsive`, the image will scale the dimensions down for smaller viewports and scale up for larger viewports.
+
+When `fill`, the image will stretch both width and height to the dimensions of the parent element, usually paired with [object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit).
+
+Defaults to `intrinsic`.
```

Comment:
Let's move the default up to the top

Reply:
Note: It'll be better to have it in the same first paragraph instead of it being at the top as a single paragraph so a result in search shows a proper description.

Reply:
Moved it to the top in the first pagraph